prow/cmd/mkpj/BUILD.bazel

```diff
@@ -9,8 +9,14 @@ load(
 
 go_binary(
     name = "mkpj",
```

Comment:
Ideally the `mkpj` target is just `mkpj`. Can we use a separate target for injecting the cncf config?

Reply:
sure